pkg/jx/cmd/get_vault_config.go

```diff
@@ -73,15 +74,21 @@ func NewCmdGetVaultConfig(f Factory, in terminal.FileReader, out terminal.FileWr
 
 // Run implements the command
 func (o *GetVaultConfigOptions) Run() error {
-	client, err := o.Factory.GetVaultClient(o.Name, o.Namespace)
+	var vaultClient vault.Client
+	var err error
+	if o.Name != "" && o.Namespace != "" {
```

Comment:
What if only one of these is empty? eg, if `name` is populated but `namespace` isn't, should we use the default namespace?

Reply:
We can handle this case instead of defaulting to system vault.